server_test.go

```diff
@@ -1506,6 +1506,37 @@ func TestServer_LimitAndOffset(t *testing.T) {
 	}
 }
 
+// Ensure the server can execute a group by with a fill(0) and very large limit
+// that results in a grouping of over 100,000 points returns an error
+func TestServer_ExecuteGroupByFillLimit(t *testing.T) {
```

Comment:
I think `server_integration_test.go` would be a better place for this. Easier to just add this to the table tests.